timeseries/src/autogluon/timeseries/models/presets.py

```diff
@@ -40,20 +40,20 @@
 )
 DEFAULT_MODEL_NAMES = {v: k for k, v in MODEL_TYPES.items()}
 DEFAULT_MODEL_PRIORITY = dict(
-    MQCNN=40,
-    MQRNN=40,
-    SimpleFeedForward=50,
-    Transformer=40,
-    TemporalFusionTransformer=45,
-    DeepAR=50,
+    MQCNN=30,
+    MQRNN=30,
+    SimpleFeedForward=45,
+    Transformer=30,
+    TemporalFusionTransformer=20,
```

Comment:
why is TFT lower than Transformer?

Reply:
TFT takes much longer to train, so I lowered it priority. I would anyway double-check the presets later, the purpose of the changes here was to "squeeze in" the AG-T model between the faster & slower methods.